src/iquest.cpp

```diff
@@ -4,6 +4,7 @@
 #include <irods/parseCommandLine.h>
 #include <irods/rcMisc.h>
 #include <irods/rodsClient.h>
+#include <irods/rodsError.h>
```

Comment:
Is this needed?

Reply:
Yes. I had inserted a call to `printErrorStack` without including its header.